src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntry.Entry.V7.Tests.cs

```diff
@@ -9,32 +9,6 @@ namespace System.Formats.Tar.Tests
     // Tests specific to V7 format.
     public class TarWriter_WriteEntry_V7_Tests : TarTestsBase
     {
-        [Fact]
-        public void ThrowIf_WriteEntry_UnsupportedFile()
```

Comment:
Obsolete test since any archive can contain entries on any format, not just one format.